app/Models/Hotel.php

```diff
@@ -21,6 +21,8 @@ class Hotel extends Model
         'description',
         'physical_address',
         'codename',
+        'location',
+        'location_description',
```

Comment:
We have no such columns in the table. They are coming from locations table. Better remove